src/templates/XmlRenderer/xmlRenderer.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { XMLRenderer } from "./template";
+import { XMLRendererSampleData } from "./sample";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import React from "react";
+
+describe("xml renderer", () => {
+  it("should render with data", () => {
+    const { getByText } = render(<XMLRenderer document={XMLRendererSampleData} handleObfuscation={() => void 0} />);
+    expect(getByText("2.1")).toBeInTheDocument();
+    expect(
+      getByText("urn:cen.eu:en16931:2017#conformant#urn:fdc:peppol.eu:2017:poacc:billing:international:sg:3.0")
```

Comment:
do you happen to understand what this line of text is? just asking

Reply:
I don't understand the purpose of the string. I got confused with this as well until I compare with the expected output HTML and found out they just place the whole string there.

Reply:
sure, must be some trade lingo thing that we dont care